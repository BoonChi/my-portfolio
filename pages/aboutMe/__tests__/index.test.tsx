import { render } from '@testing-library/react'
import { shallow } from 'enzyme'
import AboutMe from '../index'

describe('About me', () => {
  it('renders About me page unchanged', () => {
    const { container } = render(<AboutMe />)
    expect(container).toMatchSnapshot()
  })

  it('renders and reads H1 text', () => {
    const wrapper = shallow(<AboutMe />)

    const heading = <h1>About Chi</h1>

    expect(wrapper.contains(heading)).toEqual(true)
  })
})
