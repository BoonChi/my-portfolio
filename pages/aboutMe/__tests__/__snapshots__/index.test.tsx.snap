// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About me renders About me page unchanged 1`] = `
<div>
  <div
    class="main"
  >
    <h1>
      About Chi
    </h1>
  </div>
</div>
`;

exports[`About me renders About mepage unchanged 1`] = `
<div>
  <div
    class="main"
  >
    <h1>
      About Chi
    </h1>
  </div>
</div>
`;
